### PR TITLE
[Zero-Dim] elementwise: onednn: support zero dimension inputs

### PR DESCRIPTION
**Pull request by xinyu-intel**
### PR types
New features

### PR changes
OPs

### Describe
partial impl for https://github.com/PaddlePaddle/Paddle/issues/51364

### CODE REVIEW

**Comment by xinyu-intel** [comment]
@zhouwei25 @yangqun1 Please help take a review. Thanks:)

File: python/paddle/fluid/tests/unittests/mkldnn/test_elementwise_div_mkldnn_op.py [hunk 1]
```diff
@@ -111,6 +111,45 @@ def test_check_grad_ignore_x(self):
         pass
 
 
+class TestMKLDNNElementwiseDivOpZeroDim(TestMKLDNNElementwiseDivOp):
```
Comment (zhwesky2010) [inline python/paddle/fluid/tests/unittests/mkldnn/test_elementwise_div_mkldnn_op.py:114]: whether it need to check backward in Div/Mul/Sub OP?

**Review by zhwesky2010 — COMMENTED** [review]
I have no other problem, LGTM

File: python/paddle/fluid/tests/unittests/mkldnn/test_elementwise_div_mkldnn_op.py (hunk 1 above)
Comment (xinyu-intel) [reply python/paddle/fluid/tests/unittests/mkldnn/test_elementwise_div_mkldnn_op.py:114]: Hi, it seems original div/mul/sub cases with broadcast cannot pass with oneDNN. So I skip them temprarily. I'll check the broadcast in the next patch:)

Comment (zhwesky2010) [reply python/paddle/fluid/tests/unittests/mkldnn/test_elementwise_div_mkldnn_op.py:114]: OK, I seed grad is checked when no broadcast

**Review by zhwesky2010 — DISMISSED** [review]

**Review by YangQun1 — DISMISSED** [review]

**Review by zhwesky2010 — APPROVED** [review]

**Review by YangQun1 — APPROVED** [review]

**Review by jczaja — APPROVED** [review]
LGTM

**Pull request merged**